product/util.py

```diff
@@ -303,7 +303,7 @@ def greatest_references(references: typing.Iterable[DependencyBase]):
 
 def _enumerate_images(
     component_descriptor: ComponentDescriptor,
-    image_reference_filter=lambda _: True,
+    image_reference_filter=lambda _,__: True,
```

Comment:
please choose named argument names. double-check you updated all usages of _enumerate_images

--> also check `lssd`

also, consider defining a function `match_any(*args, **kwargs): return True` and pass it as default value. define locally first.